Xamarin.Forms.Core/Profiler.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Internal
+{
+	[EditorBrowsable(EditorBrowsableState.Never)]
+	public struct Profile : IDisposable
+	{
+		const int Capacity = 1000;
+		const string PROFILE = "PROFILE";
+		public struct Datum
+		{
+			public string Name;
+			public string Id;
+			public long Ticks;
+			public int Depth;
+			public int Line;
+		}
+		public static List<Datum> Data = new List<Datum>(Capacity);
+
+		static Stack<Profile> s_stack = new Stack<Profile>(Capacity);
+		static int s_depth = 0;
+		static bool s_stopped = false;
+		static Stopwatch s_stopwatch = new Stopwatch();
+
+		readonly long start;
```

Comment:
Any reason you're not using the `_` prefix for these?

Reply:
Habit. I'll refresh the conventions to be consistent. Thx!

Reply:
@kingces95 can you address this

Reply:
Done.

Reply:
I thought he was talking about the statics. I'll switch the statics to caps and instance to underbars.